config/201-clusterrole.yaml

```diff
@@ -86,3 +86,10 @@ rules:
   resources:
   - events
   verbs: *everything
+
+# Brokers read
```

Comment:
Put this next to Channels read?

Reply:
you mean this?

```
# Channels and Brokers read
- apiGroups:
  - eventing.knative.dev
  resources:
  - channels
  - brokers
  verbs: &readOnly
  - get
  - list
  - watch
```

or just as a different block below channels?

Reply:
Done